src/applications/vaos/tests/e2e/covid19-vaccine.cypress.spec.js

```diff
@@ -13,12 +13,17 @@ describe('VAOS COVID-19 vaccine appointment flow', () => {
     cy.injectAxe();
     cy.get('.va-modal-body button').click();
     cy.findAllByRole('tab').should('exist');
-    // Select COVID-19 vaccine appointment type
-    cy.get('#schedule-new-appointment-1').click();
```

Comment:
Instead of radio buttons, we need to click through the CTA, then the type of care page